ffmpeg/lpms_ffmpeg.c

```diff
@@ -1198,20 +1308,25 @@ int transcode(struct transcode_thread *h,
       // width / height will be zero for pure streamcopy (no decoding)
       decoded_results->frames += dframe->width && dframe->height;
       decoded_results->pixels += dframe->width * dframe->height;
+      has_frame = has_frame && dframe->width && dframe->height;
+      if (has_frame) last_frame = ictx->last_frame_v;
+    } else if (AVMEDIA_TYPE_AUDIO == ist->codecpar->codec_type) {
+      has_frame = has_frame && dframe->nb_samples;
+      if (has_frame) last_frame = ictx->last_frame_a;
+    }
       if (has_frame) {
```

Comment:
Can we fix indentation here?

Reply:
Fixed in https://github.com/livepeer/lpms/commit/33c7a90debed18b0eaf6da6d02c9538d54961668